### PR TITLE
Add CSVGN1 Dynamic counterpart

### PR DESCRIPTION
**Pull request by luisdi97**
Hello, I'm adding the necessary code to run dynamic simulations with the CVSGN1 model. I have a pull request in the PowerSystems repository to add the corresponding Struct. I needed a 2nd Order Lead-Lag Block with non-windup limits, so I also added it.

### CODE REVIEW

**Comment by jd-lara** [comment]
@luisdi97 you can update the dependency for PowerSystems to 2.1 to add the new struct. This PR is also missing testing. Unfortunately we don't have a reference case with this device to make the test ourselves.

**Comment by jd-lara** [comment]
@rodrigomha and @luisdi97 were we able to make a case for testing? At least we should have a case that runs even if we can't compare with PSSe at the moment.

File: src/initialization/init_device.jl [hunk 1]
```diff
@@ -417,7 +417,7 @@ function initialize_dynamic_device!(
     Rbase = Mbase
 
     # PowerFlow Data
-    Q0 = PSY.get_reactive_power(device) * Sbase / Mbase # in pu (machine base)
+    Q0 = PSY.get_reactive_power(device) # in pu (system base)
```
Comment (jd-lara) [inline src/initialization/init_device.jl:420]: is the assumption that the device is always in system base?

**Review by jd-lara — COMMENTED** [review]
The PR looks good to me. I have a outstanding question about the MVA Base

File: src/initialization/init_device.jl (hunk 1 above)
Comment (luisdi97) [reply src/initialization/init_device.jl:420]: At first I assumed the device used the machine base internally I so converted P and Q from system base to machine base, used it internally and then converted the output current from machine base to system base. But the results didn't correspond with PSS(R)E so I deleted the aforementioned convertions and then the results correponded perfectly.

**Comment by jd-lara** [comment]
@luisdi97 you are correct. That was my own mental fart about the base conversions in PSY

File: src/initialization/init_device.jl (hunk 1 above)
Comment (rodrigomha) [reply src/initialization/init_device.jl:420]: This is a larger problem on our side I think. If you use: let's say a Generator for this, then the `get_reactive_power` function will return in the device base. But using a Source or a Load will return in system base. So it looks good in your PR, assuming that you are attaching the CSVGN to a Load or a Source.

**Review by rodrigomha — APPROVED** [review]
I will merge to fix the formatter

**Pull request merged**